### PR TITLE
fix some docs 

### PR DESCRIPTION
**Pull request by TCChenlong**
### PR types

 Bug fixes
### PR changes

Docs
### Describe

paddle.compat.to_text & paddle.compat.to_bytes 显示bug
image
image
paddle.static.nn.spectral_norm
image

paddle.nn.SpectralNorm  示例代码bug
image

paddle.static.nn.group_norm 示例代码bug
image

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -3576,26 +3576,16 @@ def group_norm(input,
...
        .. code-block:: python
 
-            import paddle.fluid as fluid
             import paddle
             paddle.enable_static()
             
-            data = fluid.data(name='data', shape=[None, 8, 32, 32], dtype='float32')
-            x = fluid.layers.group_norm(input=data, groups=4)
+            data = paddle.static.data(name='data', shape=[2, 8, 32, 32], dtype='float32')
+            x = paddle.static.nn.group_norm(input=data, groups=4)
```
Comment (swtkiwi) [inline python/paddle/fluid/layers/nn.py:3588]: 这里要不要print下x，注释下输出结果呢？

**Review by swtkiwi — DISMISSED** [review]
LGTM

**Review by T8T9 — DISMISSED** [review]

**Comment by T8T9** [comment]
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by Heeenrrry — APPROVED** [review]
LGTM

**Review by zhiqiu — APPROVED** [review]

**Pull request merged**